diff --git a/node_modules/rnx-gradient/react-native-gradient.podspec b/node_modules/rnx-gradient/react-native-gradient.podspec
index 0d64732..90bdbb1 100644
--- a/node_modules/rnx-gradient/react-native-gradient.podspec
+++ b/node_modules/rnx-gradient/react-native-gradient.podspec
@@ -2,7 +2,7 @@ require "json"
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
-folly_version = '2021.06.28.00-v2'
+folly_version = '2021.07.22.00'
 folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
 
 Pod::Spec.new do |s|
